### PR TITLE
PoC for interactions with Mediator

### PR DESCRIPTION
**Pull request by rodolfomiranda**
This is the first proof of concept code that interacts with the mediator. There's an example that show how two wallets can have a conversation sending Basic Messages via the Mediator.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -256,8 +256,10 @@ dependencies {
         exclude group: 'com.soywiz.korlibs.krypto', module: 'krypto-jvm'
         exclude group: 'com.google.protobuf', module: 'protobuf-javalite'
         exclude group: 'io.ipfs.multibase.Base16'
+        exclude group: 'org.bouncycastle', module: 'bcprov-jdk15to18'
+        exclude group: 'org.jetbrains.kotlinx', module: 'kotlinx-coroutines-core'
     }
-    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.2")
+    //implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.2")
```
Comment (2byrds) [inline android/app/build.gradle:262]: I had to uncomment the kotlinx-serialization-json lib, PrismModule wasn't compiling without it.  Can you double check this on your side?

Comment (2byrds) [reply android/app/build.gradle:262]: Everything else looks good!

**Review by 2byrds — CHANGES_REQUESTED** [review]

File: android/app/build.gradle (hunk 1 above)
Comment (rodolfomiranda) [reply android/app/build.gradle:262]: if I uncomment it, I receive "More than one file was found with OS independent path 'META-INF/kotlinx-serialization-json.kotlin_module'.". Weird.

Comment (2byrds) [reply android/app/build.gradle:262]: I tried 'clean' and 'cleanBuildCache' from my IDE, and then 'build'.  and still got the build error.

I also tried from the command line:
rm -Rf node_modules;yarn cache clean;yarn install
and
yarn android

and got the same build error.
Did you try those?

**Pull request merged**